db/migrate/20230628131123_remove_product_permalink.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveProductPermalink < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :spree_products, :permalink
```

Comment:
Does it need to be reversible?

Reply:
Difficult to tell upfront. In development, I usually always test the rollback and it can be handy on staging as well. In production, it most likely doesn't need to be reversible. I just think that it's good practice. It also makes it clearer what is deleted without looking into the schema change.